### ISSUE TITLE
training gradient norm

### ISSUE THREAD

**Issue by pizzaball** [pytorch/torchtitan#1500]
### Bug description

I try to train llama model on 4 gpus and in logs I see that there is a lot of steps where grad_norm > 1 despite of the fact that in llama toml config max_norm = 1.0. Is it okay because of distributed training or not?

### Versions

I tried it using last version of torch titan

**Comment by tianyu-l** [comment]
wouldn't the total norm after clipping being `max(grad_norm, max_norm)`?

**Comment by pizzaball** [comment]
> wouldn't the total norm after clipping being `max(grad_norm, max_norm)`?

Looks like that, but as I understand tensor board logs grad_norm before clipping like max(grad_norm, max_norm)

**Comment by tianyu-l** [comment]
it's little value, so we probably won't do this
